evcouplings/compare/sifts.py

```diff
@@ -137,8 +140,37 @@ def find_homologs_jackhmmer(**kwargs):
     if config["prefix"] is None:
         config["prefix"] = path.join(tempdir(), "compare")
 
-    # run jackhmmer against sequence database
-    ar = jackhmmer_search(**config)
+    check_required(
+        config,
+        [
+            "database", "prefix", "use_bitscores",
+            "domain_threshold", "sequence_threshold",
+            "nobias", "cpu",
+        ]
+    )
+
+    # run hmmsearch (possibly preceded by hmmbuild)
+    if not use_jackhmmer:
+        hmmsearch_result = hmmbuild_and_search(
```

Comment:
should be hmmsearch_result = hmmbuild_and_search(**config), the basic idea is that the configurations can be passed into functions using the **kwargs mechanism without putting down all the parameters like this